### PR TITLE
Update payloads_not_blocked to host_not_blocked in report_generator.py

### LINKED ISSUE #65: Change Logic Behind # of Payloads Executed
# 🐛 Summary #

When a report is generated, the value attached to "payloads executed and connected to the CISA team's command-and-control C2 server (Not Blocked)." counts both border protection and host protection Not Blocked's, when it should only count the host protection Not Blocked's.

## To reproduce ##

1. Generate a test report
2. Scroll to the section "Phishing Controls Assessment"
3. Locate value associated with total payloads executed and compare with all Not Blocked's in the Host Protection Column of the chart below

## Expected behavior ##

What did you expect to happen that didn't?

We expected the total payloads executed value to match the total number of Not Blocked's in the Host Protection Column of the payloads.

## Any helpful log output or screenshots ##

Paste the results here:

Contact PCA Analysts for more information

Add any screenshots of the problem here.

### PR DESCRIPTION
**Pull request by ameliav**
#  #

## 🗣 Description ##

Update payloads_not_blocked to host_not_blocked in report_generator.py

## 💭 Motivation and context ##

"We expected the total payloads executed value to match the total number of Not Blocked's in the Host Protection Column of the payloads." Found in issue #65 

## 🧪 Testing ##

I created the report a few times with different input values to determine that the result matched.

## 📷 Screenshots ##

The screenshot show that it is only capturing the total amount from host protection.
image

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] *All* future TODOs are captured in issues, which are referenced
      in code comments.
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [x] Tests have been added and/or modified to cover the changes in this PR.
- [x] All new and existing tests pass.

## ✅ Pre-merge checklist ##

- [x] Revert dependencies to default branches.
- [ ] Finalize version.

## ✅ Post-merge checklist ##

- [ ] Create a release.

### CODE REVIEW

**Review by schmelz21 — APPROVED** [review]
Approved.
@JCantu248  or @nickviola are you able to take a quick look?

**Comment by schmelz21** [comment]
FYI - as of now current report backlog  = 8

**Comment by schmelz21** [comment]
@dav3r or @jsf9k - either one of you able to take a look? 
It is a minor calculation change, and we have some reports on hold until we update. Thanks.

**Review by nickviola — APPROVED** [review]
LGTM

File: src/tpt_reports/_version.py
Comment (dav3r) [inline src/tpt_reports/_version.py:2]: Sorry I'm a little late to the party here:

This is a big version jump for this extremely minor change.  Also, the version was never finalized prior to merging (i.e. it should have been finalized to `1.1.0`).  

I'd have expected this small PR to be only be a `patch` update putting the version to `1.0.1`.  Not a big deal now since it's already been merged, but the next PR for this repo should definitely include a finalized release version before being merged (this is precisely why we have the "Finalize version" checkbox in the PR description).

Comment (dav3r) [reply src/tpt_reports/_version.py:2]: I see that @ameliav finalized the version in https://github.com/cisagov/tpt-reports/pull/68, so good job there. 👍

**Pull request merged**
